packages/design-system/src/components/ChoiceList/ChoiceList.test.tsx

```diff
@@ -81,6 +82,21 @@ describe('ChoiceList', () => {
       expect(fieldsetEl).toBeDefined();
     });
 
+    it('allows for modification of aria attributes', () => {
+      const choices = generateChoices(4, {
+        'aria-live': 'off',
+        'aria-relevant': 'text',
+        'aria-atomic': 'true',
+      });
+      choices[0].checked = true;
+      const { container } = renderChoiceList({ choices });
+      const wrapper = container.querySelector(':nth-child(3)');
+      expect(wrapper).toHaveAttribute('aria-live', 'off');
+      expect(wrapper).toHaveAttribute('aria-relevant', 'text');
+      expect(wrapper).toHaveAttribute('aria-atomic', 'true');
+      expect(wrapper).toMatchSnapshot();
+    });
```

Comment:
We should probably also test the case where it would have applied them automatically (when `checkedChildren` is defined)

Reply:
Oh, yes good point, I thought that's what checking the first choice did mistakenly, gotta add children!

Reply:
Done.